src/main/java/io/strimzi/kafka/bridge/Application.java

```diff
@@ -23,28 +35,77 @@
+                    ar.result().getString(HttpConfig.HTTP_HOST, HttpConfig.DEFAULT_HOST),
+                    ar.result().getInteger(HttpConfig.HTTP_PORT, HttpConfig.DEFAULT_PORT));
+
+            KafkaConsumerConfig kafkaConsumerConfig = new KafkaConsumerConfig(ar.result().getString(KafkaConsumerConfig.KAFKA_CONSUMER_AUTO_OFFSET_RESET, KafkaConsumerConfig.DEFAULT_AUTO_OFFSET_RESET));
+            KafkaProducerConfig kafkaProducerConfig = new KafkaProducerConfig(Integer.toString(ar.result().getInteger(KafkaProducerConfig.KAFKA_PRODUCER_ACKS, Integer.parseInt(KafkaProducerConfig.DEFAULT_ACKS))));
+
+            KafkaConfig kafkaConfig = new KafkaConfig(ar.result().getString(KafkaConfig.KAFKA_BOOTSTRAP_SERVERS, KafkaConfig.DEFAULT_BOOTSTRAP_SERVERS),
+                    kafkaProducerConfig, kafkaConsumerConfig);
+
+            AmqpBridgeConfig amqpBridgeConfig = new AmqpBridgeConfig(kafkaConfig, amqpConfig);
+            if (amqpBridgeConfig.getEndpointConfig().isEnabled()) {
+                AmqpBridge amqpBridge = new AmqpBridge(amqpBridgeConfig);
+                vertx.deployVerticle(amqpBridge, done -> {
+                    if (done.succeeded()) {
+                        log.debug("AMQP verticle instance deployed [{}]", done.result());
+                    } else {
+                        log.debug("Failed to deploy AMQP verticle instance", done.cause());
+                    }
+                });
+            }
+
+            HttpBridgeConfig httpBridgeConfig = new HttpBridgeConfig(kafkaConfig, httpConfig);
+            if (httpBridgeConfig.getEndpointConfig().isEnabled()) {
+                HttpBridge httpBridge = new HttpBridge(httpBridgeConfig);
+                vertx.deployVerticle(httpBridge, done -> {
+                    if (done.succeeded()) {
+                        log.debug("HTTP verticle instance deployed [{}]", done.result());
+                    } else {
+                        log.debug("Failed to deploy HTTP verticle instance", done.cause());
+                    }
+                });
+            }
+        });
+    }
 
-        HttpBridgeConfig httpBridgeConfig = HttpBridgeConfig.fromMap(System.getenv());
-        if (httpBridgeConfig.getEndpointConfig().isEnabled()) {
-            HttpBridge httpBridge = new HttpBridge(httpBridgeConfig);
-            vertx.deployVerticle(httpBridge, done -> {
-                if (done.succeeded()) {
-                    log.debug("HTTP verticle instance deployed [{}]", done.result());
-                } else {
-                    log.debug("Failed to deploy HTTP verticle instance", done.cause());
-                }
-            });
+    private static String getFilePath(String arg) {
+        String[] confFileArg = arg.split("=");
+        String path = "";
+        if (confFileArg[0].equals("--config-file")) {
+            if (confFileArg[1].startsWith(File.separator)) {
```

Comment:
I guess that this will not work ... absolute path on Windows will start with something like `N:\`.